.github/README.md

```diff
@@ -6,15 +6,19 @@ A good description is clear, short, and to the point. Describe the purpose of th
 
 Describe how to install the project locally then **DELETE THIS LINE**. For example:
 
-1. `git clone git@github.com:qctrl/[REPOSITORY].git`
-1. `cd [REPOSITORY]`
-1. `install...`
+```cmd
```

Comment:
@tysonwolker I like where this is going. I'd like to know more about the decision to use `cmd`.

- Look at [GitHub Syntax Highlighting](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting).
- This links to [the languages YAML file](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml).
- Note that `cmd` denotes a [Batchfile](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L503).
- Note that `sh` denotes a [Shell](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L6000).
- See also this [Markdown code styleguide](https://arcticicestudio.github.io/styleguide-markdown/rules/code.html#escape-newlines-in-terminal-commands) - which I'm not saying we should adopt - but it's a point of reference.

So where did the choice of `cmd` come from?

Reply:
@robertlove thanks [this](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml) was a good share. Agreed `sh` is definitely the right type.

Reply:
@tysonwolker super! WDYT about using the full version `shell` rather than the alias `sh`?

Reply:
The Markdown code styleguide looks like a great resource.